src/Bicep.LangServer/Handlers/BicepDecompileParamsCommandHandler.cs

```diff
@@ -50,20 +47,23 @@ public class BicepDecompileParamsCommandHandler : ExecuteTypedResponseCommandHan
             this.bicepparamDecompiler = bicepparamDecompiler;
         }
 
-        public override Task<BicepDecompileParamsCommandResult> Handle(BicepDecompileParamsCommandParams parameters, CancellationToken cancellationToken)
+        public override async Task<BicepDecompileParamsCommandResult> Handle(BicepDecompileParamsCommandParams parameters, CancellationToken cancellationToken)
         {
+            await Task.Yield();
```

Comment:
Is this meant to prevent saturating the event loop (like `Thread.yield()` in Java or `pthread_yield()`)?

Reply:
No, it's just there to avoid having to write pseudo-async code with `Task.FromResult(...)` (Roslyn doesn't like it if you have an `async Task` method that doesn't contain any `await` statements).